VTEX - Pick and Pack API.json

```diff
@@ -0,0 +1,4196 @@
+                                    "code": "SHR0007B",
+                                    "message": "You do not have access to this order [1481120515528-01]",
+                                    "date": "2024-12-04T01:03:29.495Z",
+                                    "requestId": "3d292319-2129-47b6-a748-92a8be02576a"
+                                }
+                            }
+                        }
+                    }
+                },
+                "servers": [
+                    {
+                        "url": "https://api.pick-and-pack.com/prod/v1/{environment}",
+                        "variables": {
+                            "environment": {
+                                "default": "DefaultParameterValue"
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        "/orders/{orderId}/tracking": {
+            "patch": {
+                "tags": [
+                    "Orders"
+                ],
+                "summary": "Set order delivery",
+                "description": "This endpoint finalizes the Pick and Pack workflow by marking the order as delivered. Additionally, it allows you to update the OMS with shipping details. All fields are optional. For more information, see [Update order tracking status](https://developers.vtex.com/docs/api-reference/orders-api#put-/api/oms/pvt/orders/-orderId-/invoice/-invoiceNumber-/tracking) endpoint.\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| OMS | OMS access | **View order** |\r\n| OMS | OMS access | **Only show orders created by the user (via call center)** |\r\n| Catalog | Telesales | **Assisted Sales** |\r\n\r\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint. To learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication).\r\n\r\n>\u2757 To prevent integrations from having excessive permissions, consider the [best practices for managing app keys](https://help.vtex.com/en/tutorial/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",

```

Comment:
Yes, there are aplicable predefined roles. Please fix the text, including the apropriate predefined roles.

In fact, it sounds very strange to me the resources `View order` and `Only show orders created by the user (via call center)` would allow the user to edit order information. Do you think it is worth double checking with the team?